.github/workflows/test.yml

```diff
@@ -0,0 +1,27 @@
+name: Testing
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@master
+    - uses: actions/setup-node@v1
+    - uses: actions/setup-go@v2
+      with:
+        go-version: 1.14.x
+    - uses: actions/setup-ruby@v1
+    - name: Cache Rubygems and Node Packages
```

Comment:
Does this work for multiple files? Wouldn't it be better to cache once for each dependency type?

Reply:
Yeah, fair enough. It does work, but probably better to cache against each dependency type, as you say